src/cs-core/tools/Mark.cpp

```diff
@@ -73,13 +73,12 @@ void main()
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
 Mark::Mark(std::shared_ptr<InputManager> const& pInputManager,
-    std::shared_ptr<SolarSystem> const&         pSolarSystem,
-    std::shared_ptr<GraphicsEngine> const&      graphicsEngine,
+    std::shared_ptr<SolarSystem> const& pSolarSystem, std::shared_ptr<Settings> const& settings,
     std::shared_ptr<TimeControl> const& pTimeControl, std::string const& sCenter,
     std::string const& sFrame)
     : mInputManager(pInputManager)
     , mSolarSystem(pSolarSystem)
-    , mGraphicsEngine(graphicsEngine)
+    , mSettings(settings)
     , mTimeControl(pTimeControl)
```

Comment:
Pass by value and move.

Reply:
You're right but in order to avoid too many conflicts with the `feature/clang-tidy` I will do this after it got merged.